gobblin-docs/user-guide/Docker-Integration.md

```diff
@@ -12,21 +12,25 @@ For more information on Docker, including how to install it, check out the docum
 
 # Docker Repositories
 
-Gobblin currently has four different repositories, and all are on Docker Hub [here](https://hub.docker.com/u/gobblin/). We are also starting to use [Apache's repository](https://hub.docker.com/r/apache/gobblin/tags?page=1&ordering=last_updated) for our images. 
+Github Actions pushes the latest docker image to the Apache DockerHub repository [here](https://hub.docker.com/r/apache/gobblin) from `gobblin-docker/gobblin/alpine-gobblin-latest/Dockerfile`
 
-The `gobblin/gobblin-wikipedia` repository contains images that run the Gobblin Wikipedia job found in the [getting started guide](../Getting-Started). These images are useful for users new to Docker or Gobblin, they primarily act as a "Hello World" example for the Gobblin Docker integration.
+To run this image, you will need to pass in the corresponding execution mode.
```

Comment:
We can add a reference for the execution mode:
https://gobblin.readthedocs.io/en/latest/user-guide/Gobblin-Deployment/
or
https://github.com/apache/incubator-gobblin/blob/master/gobblin-docs/user-guide/Gobblin-Deployment.md

Reply:
Good suggestion, thanks! These docs are a little out of date so hopefully we can update them soon :x